adyenv6b2ccheckoutaddon/acceleratoraddon/web/src/com/adyen/v6/controllers/pages/AdyenSummaryCheckoutStepController.java

```diff
@@ -607,7 +608,8 @@ private String redirectToOrderSummaryWithError(final Model model,
                                                    final RedirectAttributes redirectAttributes,
                                                    final String messageKey) throws CommerceCartModificationException, CMSItemNotFoundException {
         final CartData cartData = getCheckoutFacade().getCheckoutCart();
-        if(cartData == null || cartData.getAdyenPaymentMethod() == null) {
+        if(cartData == null || cartData.getAdyenPaymentMethod() == null
+                || PAYMENT_METHOD_APPLE_PAY.equals(cartData.getAdyenPaymentMethod())) {
```

Comment:
minor: to keep it consistent with other namings, PAYMENT_METHOD_APPLE_PAY should be PAYMENT_METHOD_APPLEPAY